docs/diagrams/TotalOrdersSequenceDiagram2.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
+!include style.puml
+
+box UI UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+participant ":TotalOrdersWindow" as TotalOrdersWindow UI_COLOR
+end box
+
+box Logic LOGIC_COLOR_T1
+participant ":Logic" as Logic LOGIC_COLOR
+end box
+
+[-> MainWindow : executeCommand("totalorders")
+activate MainWindow
+
+MainWindow -> Logic : execute("totalorders")
+activate Logic
+
+Logic --> MainWindow
+deactivate Logic
+
+MainWindow -> MainWindow : handleTotalOrders()
+activate MainWindow
+
+MainWindow -> TotalOrdersWindow : reloadData()
+activate TotalOrdersWindow
+
+TotalOrdersWindow -> Logic : getClientTotalOrders()
+activate Logic
+
+Logic --> TotalOrdersWindow
+deactivate Logic
+
```

Comment:
I think there should be a 'deactivate' process here too.
What's wrong: 
The activation bar of the main window still keeps going after the method returns.

Reply:
Thanks for pointing out!